src/ext/zcache/functions.zsh

```diff
@@ -62,10 +62,16 @@
 #   Nothing. Generates _ZCACHE_META_PATH and _ZCACHE_PAYLOAD_PATH
 -zcache-generate-cache () {
   local -aU _extensions_paths
+  local -aU _autoloaded_functions
   local -a _bundles_meta
   local _payload=''
+  local _sourcing_payload=''
   local location
 
+  local pfunction_glob='^([_.]*|prompt_*_setup|README*|*~)(-.N:t)'
```

Comment:
I'm not really sure the best way to move this (and the loop) into a function, as they do similar but different things.